app/models/member.rb

```diff
@@ -32,6 +32,7 @@ class Member < ActiveRecord::Base
   scope :confirmed, -> { where('confirmed_at IS NOT NULL') }
   scope :located, -> { where("location <> '' and latitude IS NOT NULL and longitude IS NOT NULL") }
   scope :recently_signed_in, -> { reorder('updated_at DESC') }
+  scope :recently_joined, -> { reorder("created_at desc") }
```

Comment:
I think this should probably be confirmed_at so we get the time when they actually click the confirmation link rather than when they first sign up. I'm open to argument on this though!